Web/scripts/fluid_properties.IF97Conformance.py

```
@@ -0,0 +1,447 @@
+                    # below.
+                    pass
+            t0 = time.perf_counter()
+            ok = 0
+            for k in range(N):
+                try:
+                    CP.PropsSI(prop, 'T', T_arr[k], 'P', p_arr[k], backend)
+                    ok += 1
+                except Exception:
+                    # Sample outside the backend's validity envelope;
+                    # don't count it toward the per-call mean.
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)